src/main/java/com/amogoscode/groupe/ebankingsuite/User/User.java

```diff
@@ -0,0 +1,136 @@
+package com.amogoscode.groupe.ebankingsuite.User;
+
+import jakarta.persistence.*;
+import lombok.Getter;
+import lombok.Setter;
+import org.springframework.security.core.GrantedAuthority;
+import org.springframework.security.core.userdetails.UserDetails;
+
+import java.time.LocalDateTime;
+import java.util.Collection;
+import java.util.Objects;
+
+@Entity
+@Getter
+@Setter
+@Table(name = "users")
+public class User implements UserDetails {
+
+    @Id
+    @SequenceGenerator(
+            name = "user_id_sequence",
+            sequenceName = "user_id_sequence"
+    )
+    @GeneratedValue(
+            strategy = GenerationType.SEQUENCE,
+            generator = "user_id_sequence"
+    )
+    private Integer id;
+    @Column(nullable = false)
+    private String fullName;
+    @Column(nullable = false)
+    private String emailAddress;
+    @Column(nullable = false)
+    private String password;
+    @Column(nullable = false)
+    private boolean isNotBlocked;
+    private LocalDateTime createdAt;
+    private LocalDateTime updatedAt;
+
+    public User(String fullName, String emailAddress, String password, boolean isNotBlocked) {
+        this.fullName = fullName;
+        this.emailAddress = emailAddress;
+        this.password = password;
+        this.createdAt=LocalDateTime.now();
+        this.updatedAt=LocalDateTime.now();
+        this.isNotBlocked = isNotBlocked;
+    }
+
+    public User() {
+    }
+
+    public Integer getId() {
```

Comment:
why to use setter and getter while you using lombok ?

Reply:
This will be corrected in the subsequent commit, This was a mistake

Reply:
changes have been pushed in a new commit